### DISCUSSION TITLE
you have no power here gifz What stage am I lacking?

### DISCUSSION THREAD

**Discussion by aqilabihar** [drizzle-mizzle/CharacterAI-Discord-Bot#158] [General]
![Captures](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/60000505/b2fa2f97-fad0-42f6-bcd2-6d7c88539c4d)
what does this mean?

**Comment by drizzle-mizzle** [comment]
https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/issues/131

**Reply by drizzle-mizzle** [reply]
setting your id in hoster_discord_id might also work
